src/IECoreDelight/Renderer.cpp

```diff
@@ -327,18 +325,45 @@ class DelightOutput : public IECore::RefCounted
 				layerName = IECore::CamelCase::join( layerTokens.begin(), layerTokens.end(), IECore::CamelCase::AllExceptFirst);
 			}
 
-			ParameterList layerParams;
+			ParameterList layerParams( output->parameters() );
 
 			layerParams.add( "variablename", variableName );
 			layerParams.add( "variablesource", variableSource );
 			layerParams.add( "layertype", layerType );
 			layerParams.add( "layername", layerName );
 			layerParams.add( { "withalpha", &withAlpha, NSITypeInteger, 0, 1, 0 } );
 
-			const string scalarFormat = this->scalarFormat( output );
-			const string colorProfile = scalarFormat == "float" ? "linear" : "sRGB";
-			layerParams.add( "scalarformat", scalarFormat );
-			layerParams.add( "colorprofile", colorProfile );
+			const double filterSize = parameter<float>( output->parameters(), "filtersize", 3.0 );
```

Comment:
What's the reason for taking `filtersize` here when the NSI name is `filterwidth`?

Reply:
TBH, I don't remember at the moment :) It probably was related to me trying what would be the best way to get a Gaffer float plug converted to a double type for the NSI filterwidth attribute. I'll double check once I revisit the code for the individual smaller PRs, but I'm pretty sure we can use `filterwidth` instead of `filtersize` here.